docs/application/dotnet/guides/nui/resources.md

```diff
@@ -7,7 +7,7 @@ This tutorial describes the use of Image resources in NUI.
 
 The two basic methods of handling image resources in NUI are as follows:
  
-* `Visuals`: For more details, see [Visuals tutorial](visuals.md) page.
+1. `Visuals` are the main building block for UI components and provide reusable rendering logic that is controlled using properties. For more information, see [Visuals tutorial](visuals.md) page:
 
 ```
```

Comment:
@mihashco 

What is your opinion on the following? 

Global: Request you to align the code segments under the numbered list by 4 spaces. 

The page layout will look better this way.